web/elm/src/Resource/Resource.elm

```diff
@@ -883,6 +885,11 @@ view session model =
         ]
 
 
+tooltip : Model -> a -> Maybe Tooltip.Tooltip
+tooltip _ _ =
+    Nothing
+
+
```

Comment:
Glad that we're adding this into multiple places! We can definitely make use of `tooltip` in future.